lib/web/servers.go

```diff
@@ -157,3 +158,61 @@ func (h *Handler) getDesktopHandle(w http.ResponseWriter, r *http.Request, p htt
 	// to see the desktop once in the UI, so just take the first one.
 	return ui.MakeDesktop(windowsDesktops[0]), nil
 }
+
+// desktopIsActive checks if a desktop has an active session and returns a desktopIsActive.
+//
+// GET /v1/webapi/sites/:site/desktops/:desktopName/active
+//
+// Response body:
+//
+// {"active": bool}
+func (h *Handler) desktopIsActive(w http.ResponseWriter, r *http.Request, p httprouter.Params, ctx *SessionContext, site reversetunnel.RemoteSite) (interface{}, error) {
+	desktopName := p.ByName("desktopName")
+	trackers, err := h.auth.proxyClient.GetActiveSessionTrackersWithFilter(r.Context(), &types.SessionTrackerFilter{
+		Kind: string(types.WindowsDesktopSessionKind),
+		State: &types.NullableSessionState{
+			State: types.SessionState_SessionStateRunning,
+		},
+		DesktopName: desktopName,
+	})
+
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	checker, err := ctx.GetUserAccessChecker()
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+L:
```

Comment:
```suggestion
```

Reply:
The label is required to break out of a loop from a switch statement

https://stackoverflow.com/questions/11104085/does-a-break-statement-break-from-a-switch-select